<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMusicPlayer</class>
 <widget class="QWidget" name="IMusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMusicPlayer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_load">
           <property name="text">
            <string>加载本地音乐</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combox_play_model">
           <item>
            <property name="text">
             <string>顺序播放</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>随机播放</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>单曲循环</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>音量调节</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_volume">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_now">
        <attribute name="title">
         <string>本地音乐</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_history">
        <attribute name="title">
         <string>播放历史</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSlider" name="slider_position">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btn_prev">
           <property name="text">
            <string>上一首</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_pause">
           <property name="text">
            <string>暂停</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_next">
           <property name="text">
            <string>下一首</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_player">
           <property name="text">
            <string>播放当前选中</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
